R/fstg02.R

```diff
@@ -74,9 +74,8 @@ fstg02_main <- function(adam_db,
   execute_with_args(
     g_forest,
     tbl = result,
-    ...,
-    gp = grid::gpar(fontsize = 7),
-    draw = FALSE
+    font_size = 7,
+    ...
```

Comment:
```suggestion
    ...,
    font_size = 7
```

to allow font_size be updated through `...`